ultrasonic/build.gradle

```diff
@@ -13,6 +13,7 @@ android {
 
         minSdkVersion versions.minSdk
         targetSdkVersion versions.targetSdk
+        multiDexEnabled true
```

Comment:
Is it really needed for the app? App should be not so big to require multidex :thinking:

Reply:
After migrating to Androidx, the build was throwing errors because of more functions than 65535, and suggested to use multidex.
I'll try to test it again.

Reply:
Interesting :thinking: 

Would be interesting to see what dependency brings so many methods count. This project does not have many dependencies.

Reply:
@Tapchicoma I'm doing some updates to use gradle '6.5.0' but breaks all tests, seems that is a minor change but I cannot find the key

Reply:
Checked the number of functions with dexcount, these namespaces contain the most functions:

```
methods  fields   package/class name
5533     853      android
15440    11354    androidx
11282    3220     com.fasterxml
3159     3668     com.google.android.material
20984    4727     kotlin
4606     5833     org.moire.ultrasonic
```

Overall Androidx, Kotlin and FasterXml are responsible for most of the 69643 total functions.

Reply:
probably you could enable multidex only for debug builds. I will expect proguard will cut out most of this methods for release builds.

Probably app minimal supported sdk should be bumped to 21 to get rid of multidex at all.

@ogarcia old version of AGP could also be a problem when updating to newer Gradle.

Reply:
Yes, minSdk 21 would solve this entirely, but I didn't want to bump it up when it isn't absolutely necessary. Google says that ~94% of the devices are above or on Sdk 21.
What do you guys say, leave it (just change to debug only) or bump it?

Reply:
I prefer maintain minSdk in 14 until we have a blocked feature. Not much users with Android 4, but still we have some.

Reply:
@Tapchicoma If I update ktlint, ktlintGradle and jacoco can upgrade gradle to 6.5.0 and buil app without problems, but some test fail to execute

Reply:
Moved multidex to debug.